CONTRIBUTING.md

```diff
@@ -5,9 +5,9 @@ source projects you are aware of. It will be amazing if you could help us by doi
 
 - File an issue in [the issue tracker](https://github.com/line/armeria/issues) to report bugs and propose new features and
   improvements.
-- Ask a question by creating a new issue in [the issue tracker](https://github.com/line/armeria/issues).
-  - Browse [the list of previously answered questions](https://github.com/line/armeria/issues?q=label%3Aquestion).
+- Ask a question tagged with `armeria` at [StackOverflow](https://stackoverflow.com/questions/tagged/armeria).
```

Comment:
I think it's often hard to know what is a bug vs what is a question. Should we remove bug from the previous bullet point to encourage first contact being in SO or slack?

Reply:
Let me reorder the bullets, then, so Slack link appears first. What do you think?

Reply:
New bullets:

- Join [our Slack workspace](https://join.slack.com/t/line-armeria/shared_invite/enQtNjIxNDU1ODU1MTI2LWRlMGRjMzIwOTIzMzA2NDA1NGMwMTg2MTA3MzE4MDYyMjUxMjRlNWRiZTc1N2Q3ZGRjNDA5ZDZhZTI1NGEwODk) to chat with us.
- Ask a question tagged with `armeria` at [StackOverflow](https://stackoverflow.com/questions/tagged/armeria).
- File an issue in [the issue tracker](https://github.com/line/armeria/issues) to report bugs or suggest an idea.
- Contribute your work by sending [a pull request](https://github.com/line/armeria/pulls).